skyrim-platform/src/platform_se/skyrim_platform/GameEventSinks.cpp

```diff
@@ -1681,7 +1686,28 @@ RE::BSEventNotifyControl GameEventSinks::ProcessEvent(
   auto refId = ref ? ref->formID : 0;
   auto type = converted->type;
 
-  SkyrimPlatform::GetSingleton().AddUpdateTask([ref, refId, type] {
+  const char* eventName;
```

Comment:
theoretically uninitialized variable can be read from here. please always initialize variables.